.eslintrc

```diff
@@ -51,6 +51,13 @@
         "promise/no-new-statics": "error",
         "promise/no-return-in-finally": "error",
         "react/jsx-handler-names": "error",
+        "react/no-children-prop": "error",
+        "react/no-typos": "error",
+        "react/no-string-refs": "error",
+        "react/no-this-in-sfc": "error",
+        "react/no-unescaped-entities": "error",
+        "react/react-in-jsx-scope": "error",
+        "react/require-render-return": "error",
```

Comment:
think we should also add `react/no-direct-mutation-state`. Also, `react/sort-prop-types` may be helpful

Reply:
`react/no-direct-mutation-state` would be a solid addition.  In the past we tried `react/sort-prop-types` in webapp, but it turned out to be a lot of work for not much benefit.